src/main/java/frc/robot/Robot.java

```diff
@@ -55,17 +55,15 @@ public void robotInit() {
     public void robotPeriodic() {
         CommandScheduler.getInstance().run();
 
-        vision.frontCamGetEstimatedGlobalPose().ifPresent(estimatedPose -> {
-            driveTrain.addVisionMeasurement(estimatedPose.estimatedPose.toPose2d(),
-                    estimatedPose.timestampSeconds,
-                    vision.getEstimationStdDevs());
-        });
-
-        vision.backCamGetEstimatedGlobalPose().ifPresent(estimatedPose -> {
-            driveTrain.addVisionMeasurement(estimatedPose.estimatedPose.toPose2d(),
-                    estimatedPose.timestampSeconds,
-                    vision.getEstimationStdDevs());
-        });
+        vision.frontCamGetEstimatedGlobalPose().ifPresent(estimatedPose ->
+                driveTrain.addVisionMeasurement(estimatedPose.estimatedPose.toPose2d(),
```

Comment:
This is not functionally equivalent. The new version returns a value instead of void. Please put back